map_view/identify-layers/README.md

```diff
@@ -1,26 +1,32 @@
-# Identify Layers
+# Identify layers
 
-Get clicked features from multiple layers.
+Identify features in all layers in a map. 
 
-![](IdentifyLayers.png)
+![Image of identify layers](IdentifyLayers.png)
+
+## Use case
+
+IdentifyLayers allows users to tap on a map, returning features at that location across multiple layers. Because some layer types have sublayers, the sample recursively counts results for sublayers within each layer.
```

Comment:
```suggestion
Identify layers allows users to return features on a map at a specified location across multiple layers. Because some layer types have sublayers, the sample recursively counts results for sublayers within each layer.
```